tests/index.test.js

```diff
@@ -117,6 +117,57 @@ describe('create-an-issue', () => {
     expect(tools.log.success).toHaveBeenCalled()
   })
 
+  it('Creates a new issue when updating existing issues is enabled but no issues with the same title exist', async () => {
```

Comment:
Tiny nit, capitalization just for consistency!

```suggestion
  it('creates a new issue when updating existing issues is enabled but no issues with the same title exist', async () => {
```